evergreen/run-tests.sh

```diff
@@ -63,6 +60,10 @@ else
 fi
 
 echo "Final MongoDB_URI: $MONGODB_URI"
-echo "Final MongoDB URI with multiple mongoses: $MONGODB_URI_WITH_MULTIPLE_MONGOSES"
-for var in TMP TEMP NUGET_PACKAGES NUGET_HTTP_CACHE_PATH APPDATA; do setx $var z:\\data\\tmp; export $var=z:\\data\\tmp; done
+if [ "$TOPOLOGY" == "sharded_cluster" ]; then
+  echo "Final MongoDB URI with multiple mongoses: $MONGODB_URI_WITH_MULTIPLE_MONGOSES"
```

Comment:
Minor tweak to only print the URI with multiple ``mongos``es when it's relevant.

Reply:
Acknowledged